app/routes/pages/components/PageEditor.js

```diff
@@ -167,7 +168,10 @@ export default class PageEditor extends Component<Props, State> {
                 message="Er du sikker på at du vil slette denne infosiden?"
                 onConfirm={this.onDelete}
               >
-                <Button>Slett</Button>
+                <Button danger>
+                  <Icon name="trash" size={19} />
+                  Slett
+                </Button>
```

Comment:
Not aligned with the other button, and if its not too much to ask maybe add a little bit of padding to the inputs to match the distance between all the four elements in the line;
![image](https://user-images.githubusercontent.com/13599770/185810980-a497721b-caf4-446b-8459-c1f9a685c92c.png)